### PR TITLE
LPS-110733 Remove TinyMCE WYSIWYG Support

### PR DESCRIPTION
**Pull request by julien**

### CODE REVIEW

**Comment by julien** [comment]
ci:test:sf

**Comment by julien** [comment]
ci:test:stable

**Comment by julien** [comment]
ci:test:relevant

**Comment by julien** [comment]
@john-co, could you help me analyse the errors of `ci:test:relevant` when you have time?
thanks

File: portal-impl/src/META-INF/javadocs-all.xml [hunk 1]
```diff
@@ -229204,6 +229204,7 @@ See https://issues.liferay.com/browse/LEP-2297.
 		</field>
 		<field>
 			<name>RESOURCE_TYPE_EDITOR_TINYMCEEDITOR</name>
+			<deprecated><![CDATA[As of 7.3.0, with no direct replacement.]]></deprecated>
```
Comment (jbalsas) [inline portal-impl/src/META-INF/javadocs-all.xml]: I think the prior message was better and more informative

**Comment by jbalsas** [comment]
Hey @julien, looks like you're still missing the `.lfr-build-portal-deprecated` marker

File: portal-impl/src/META-INF/javadocs-all.xml (hunk 1 above)
Comment (julien) [reply portal-impl/src/META-INF/javadocs-all.xml]: You mean this https://github.com/julien/liferay-portal/pull/245#discussion_r398407232?

Comment (jbalsas) [reply portal-impl/src/META-INF/javadocs-all.xml]: yes

**Comment by julien** [comment]
@jbalsas you're right. I'll close this and submit a another pull request

File: portal-impl/src/META-INF/javadocs-all.xml (hunk 1 above)
Comment (julien) [reply portal-impl/src/META-INF/javadocs-all.xml]: ok :+1:

**Comment by julien** [comment]
ci:stop

**Comment by julien** [comment]
Re-submitted [here](https://github.com/julien/liferay-portal/pull/253)

**Comment by john-co** [comment]
**Reviewed. Test failures are unrelated.** :heavy_check_mark: 

**Testray Comparison**
- [unique failures](https://testray.liferay.com/home/-/testray/runs/compare?compare=true&view=details&statusA=3&testrayRunIdB=1072403748&testrayRunIdA=1073116742&statusB=2) are not related

**Comment by john-co** [comment]
The other failure for KnowledgeBase looks like an existing bug/task: https://issues.liferay.com/browse/LPS-110014

**Pull request closed without merging**